src/options.rs

```diff
@@ -164,6 +190,21 @@ impl Options {
         );
       }
     }
+
+    let mut num_tr = 0;
+    for descriptor in client.list_descriptors(None)?.descriptors {
+      let desc = descriptor.desc;
+      if Regex::new(r"tr.*").unwrap().is_match(&desc) {
```

Comment:
We can use `.starts_with("tr(")`

Reply:
Can also do:

```rust
let tr = descriptors.iter().filter(|descriptor| descriptor.starts_with("tr(")).count();
let rawtr = descriptors.iter().filter(|descriptor| descriptor.starts_with("rawtr(")).count();

if tr != 2 {
  bail!(…);
}

if descriptors.len() != 2 + rawtr {
  bail!(…);
}
```

Reply:
The error messages for both bails should be more descriptive, something like "this does not appear to be an ord wallet, please create an ord wallet using `ord wallet create`".